README.md

```diff
@@ -97,53 +180,110 @@ Create a [`views` directory](/server/views) to store server-side templates.
 $ mkdir server/views
 ```
 
-Add [server-side templating configurations to `server.js`](/server/server.js#L11-L20).
-
 Create [`index.ejs` in the views directory](/server/views/index.ejs).
 
-[Configure `server.js`](/server/server.js#L11-L20) to use server-side
+[Configure `server.js`](/server/server.js#L5-L11) to use server-side
 templating. Remember to import the [`path`](/server/server.js#L3) package.
 
+### Relpace root
+
+Replace the default `root.js` with [a new one](/server/boot/root.js) which passes a given customer's id to template engine.
```

Comment:
I don't think we need root anymore with the new layout? We should be mapping in middleware.json instead I think.